timemachine/cpp/src/kernels/k_nonbonded_precomputed.cuh

```diff
@@ -114,7 +116,7 @@ void __global__ k_nonbonded_precomputed(
 
                 if (du_dp) {
                     // du/dp
-                    g_q_ij += FLOAT_TO_FIXED_DU_DP<RealType, FIXED_EXPONENT_DU_DCHARGE>(erfc_beta * inv_dij);
+                    g_q_ij += FLOAT_TO_FIXED_DU_DP<RealType, FIXED_EXPONENT_DU_DCHARGE>(damping_factor * inv_dij);
                     g_dw_ij += FLOAT_TO_FIXED_DU_DP<RealType, FIXED_EXPONENT_DU_DW>(delta_w * force_prefactor);
```

Comment:
are the du_dps fixed now?

Reply:
@maxentile ?

Reply:
I believe this is correct as written.

(force_prefactor reflects updated implementation of du_dr, but other terms don't need to change)

https://github.com/proteneer/timemachine/blob/b0ec4ae9c3c3b4fa7ae84c0e6ffaa7c81e04e417/timemachine/cpp/src/kernels/k_nonbonded_precomputed.cuh#L103-L105

Reply:
This was briefly addressed offline.